yb-voyager/cmd/exportSchema.go

```diff
@@ -396,8 +396,7 @@ func applyShardingRecommendationIfMatching(sqlInfo *sqlInfo, shardedTables []str
 	case ORACLE:
 		// TODO: handle case-sensitivity properly
 		for _, shardedTable := range shardedTables {
-			parts := strings.Split(shardedTable, ".")
-			if strings.ToLower(parts[1]) == parsedTableName {
+			if strings.ToLower(shardedTable) == parsedTableName {
```

Comment:
can we add a comment/note here, that assuming that shardedTable will not be fully qualified in case of oracle.

Just thinking from perspective of supported multiple oracle schemas in future.